File: Javascript/backend_nest/src/infra/db/inmemory/data/AuthorDB.ts
import { AuthorInMemory } from '../entities/AuthorInMemory';

export const AuthorinMemoryDB: Record<number, AuthorInMemory> = {
  1: new AuthorInMemory(
    1,
    'Victor',
    'Hugo',
    '1802-02-26',
    'Français',
    '1885-05-22',
    'Poète, romancier et dramaturge, célèbre pour Les Misérables.'
  ),
  2: new AuthorInMemory(
    2,
    'Jane',
    'Austen',
    '1775-12-16',
    'Anglais',
    '1817-07-18',
    'Romancière britannique célèbre pour Orgueil et Préjugés.'
  ),
  3: new AuthorInMemory(
    3,
    'Mark',
    'Twain',
    '1835-11-30',
    'Américain',
    '1910-04-21',
    'Écrivain américain connu pour Les Aventures de Tom Sawyer.'
  ),
  4: new AuthorInMemory(
    4,
    'Gabriel',
    'García Márquez',
    '1927-03-06',
    'Colombien',
    '2014-04-17',
    'Écrivain colombien, prix Nobel de littérature, célèbre pour Cent ans de solitude.'
  ),
  5: new AuthorInMemory(
    5,
    'Mary',
    'Shelley',
    '1797-08-30',
    'Anglais',
    '1851-02-01',
    'Romancière britannique, célèbre pour Frankenstein.'
  ),
};

File: Python/backend_fastapi/infra/db/inmemory/data/author_db.py
from datetime import date
from infra.db.inmemory.entities.author_inmemory import AuthorInMemory

author_db = {
    1: AuthorInMemory(
        id=1,
        first_name="Victor",
        last_name="Hugo",
        birth_date=date(1802, 2, 26),
        nationality="Français",
        death_date=date(1885, 5, 22),
        biography="Écrivain et poète français, auteur de Les Misérables.",
        books=[101, 102]
    ),
    2: AuthorInMemory(
        id=2,
        first_name="George",
        last_name="Orwell",
        birth_date=date(1903, 6, 25),
        nationality="Anglais",
        death_date=date(1950, 1, 21),
        biography="Écrivain britannique, auteur de 1984 et Animal Farm.",
        books=[103, 104]
    )
}